alns/accept/GreatDeluge.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional
+
+from alns.accept.AcceptanceCriterion import AcceptanceCriterion
+
+
+class GreatDeluge(AcceptanceCriterion):
+    """
+    The Great Deluge (GD) criterion accepts solutions if the candidate solution
+    has a value lower than a threshold (originally called the water level [1]).
+    The initial threshold is computed as
+
+    ``threshold = alpha * initial.objective()``
+
+    where ``initial`` is the initial solution passed-in to ALNS, inferred
+    from the best solution at the first iteration. The threshold is updated in
+    each iteration as
+
+    ``threshold = threshold - beta * (threshold - candidate.objective()``
```

Comment:
Rephrase: this line is equivalent to the implementation, but phrased slightly differently. I suggest either changing this line, or the implementation, whichever follows the paper _exactly_.